### PR TITLE
Mark Rails `~> 6.0.4` as patched

### PR DESCRIPTION
**Pull request by orien**
### Context

I'm receiving automated vulnerability alerts for today's release of Rails 6.0.4.

I believe this version of Rails is patched against these vulnerabilities.

### Change

Mark Rails `~> 6.0.4` as patched for CVE-2021-22885, CVE-2021-22902, CVE-2021-22904, and CVE-2021-22880.

### CODE REVIEW

File: gems/actionpack/CVE-2021-22885.yml [hunk 1]
```diff
@@ -64,4 +64,5 @@ patched_versions:
   - "~> 5.2.4.6"
   - "~> 5.2.6"
   - "~> 6.0.3.7"
```
Comment (reedloden) [inline gems/actionpack/CVE-2021-22885.yml:66]: Instead, could we do:
`- "~> 6.0.3, >= 6.0.3.7"`

File: gems/actionpack/CVE-2021-22902.yml [hunk 2]
```diff
@@ -38,4 +38,5 @@ unaffected_versions:
 
 patched_versions:
   - "~> 6.0.3.7"
```
Comment (reedloden) [inline gems/actionpack/CVE-2021-22902.yml:40]: Instead, could we do:
`- "~> 6.0.3, >= 6.0.3.7"`

File: gems/actionpack/CVE-2021-22904.yml [hunk 3]
```diff
@@ -59,4 +59,5 @@ patched_versions:
   - "~> 5.2.4.6"
   - "~> 5.2.6"
   - "~> 6.0.3.7"
```
Comment (reedloden) [inline gems/actionpack/CVE-2021-22904.yml:61]: Instead, could we do:
`- "~> 6.0.3, >= 6.0.3.7"`

File: gems/activerecord/CVE-2021-22880.yml [hunk 4]
```diff
@@ -61,4 +61,5 @@ unaffected_versions:
 patched_versions:
   - "~> 5.2.4, >= 5.2.4.5"
   - "~> 6.0.3.5"
```
Comment (reedloden) [inline gems/activerecord/CVE-2021-22880.yml:63]: Instead, could we do:
`- "~> 6.0.3, >= 6.0.3.5"`

**Review by reedloden — CHANGES_REQUESTED** [review]
Thanks for submitting this! I have proposed an alternative that I think will be better. Let me know your thoughts.

File: gems/actionpack/CVE-2021-22885.yml (hunk 1 above)
Comment (orien) [reply gems/actionpack/CVE-2021-22885.yml]: 👍 Updated in [`3f198f2`](https://github.com/rubysec/ruby-advisory-db/compare/36785ac88f0c8ef4ee9ba0c6527d65694bb67a25..3f198f2b13dce1f72e03393d4d935a9cbce121fd).

**Review by reedloden — APPROVED** [review]
Thanks! Looks good!

**Pull request merged**